src/test/crimson/seastore/onode_tree/test_staged_fltree.cc

```diff
@@ -22,6 +22,38 @@
 
 using namespace crimson::os::seastore::onode;
 
+#define INTR(fun, t)            \
+  with_trans_intr(              \
+    t,                          \
+    [&] (auto &tr) {            \
+      return fun(tr);           \
+    }                           \
+  )
+
+#define INTRV(fun, t, args...)          \
+  with_trans_intr(                      \
+    t,                                  \
+    [&, args] (auto &tr) {              \
+      return fun(tr, args);             \
+    }                                   \
+  )
+
+#define INTR_R(fun, t, args...)         \
+  with_trans_intr(                      \
+    t,                                  \
+    [&] (auto &tr) {                    \
+      return fun(tr, args);             \
+    }                                   \
+  )
+
+#define INTR_WITH_PARAM(fun, c, b, v)   \
```

Comment:
How about replacing all INTR_WITH_PARAM by INTR_R and remove it?

Reply:
seems can't.  here pass c, and use c.t in with_trans_intr. and fun also use c,